kenmerken-nummeraanduiding-in-onderzoek.feature

```diff
@@ -0,0 +1,63 @@
+# language: nl
+Functionaliteit: indicatie één of meerdere kenmerken van nummeraanduiding in onderzoek
+Design decisions:
+
+1. indicatie inOnderzoek (ook individuele) niet in response opnmen als de Resource of één of meerdere kenmerken van de Resource niet in onderzoek zijn
+Reden:
+- Het opnemen van de InOnderzoek gegevensgroep bij niet in onderzoek zorgt voor onnodig grote response
+
+2. geen (lijst van) string om aan te geven welke kenmerk in onderzoek zijn
+Reden:
+- er moet dan logica worden geïmplementeerd om te achterhalen welke kenmerken in onderzoek zijn
+
+Achtergrond:
+    Gegeven BAG bevat de volgende nummeraanduidingen
+    | identificatie | in onderzoek                                         |
+    | 1             | nee                                                  |
+    | 2             | ja, alle kenmerken                                   |
+    | 3             | ja, alleen kenmerken documentDatum en documentNummer |
+
+Scenario: nummeraanduiding is niet in onderzoek

```

Comment:
Dit kan niet meer in BAG 2.0.

Reply:
Wat ik bedoel is 'geen enkele kenmerk van nummeraanduiding is in onderzoek'